.github/workflows/infrautils-microservice-s3-deploy.yml

```diff
@@ -0,0 +1,70 @@
+# This workflow installs infrautils (DevOps maintained infrastructure as service python package) from JFrog
+# and uses infrautils (boto3) to deploy a cloudformation stack update
+# directly from the java microservice template in S3
```

Comment:
this isn't from the JMT right?

Reply:
I think referencing the JMT here is confusing. it did used to live there but it doesn't anymore. it's just the cloudformation template used to deploy microservices.

Reply:
that makes sense! I've updated the description accordingly to reflect that it's a generic microservice template.